tests/cpp-tests/CMakeLists.txt

```diff
@@ -329,8 +330,7 @@ list(APPEND GAME_SOURCE
      Classes/ZwoptexTest/ZwoptexTest.cpp
      Classes/SpriteFrameCacheTest/SpriteFrameCacheTest.cpp
      Classes/controller.cpp
-     # Classes/VRTest/VRTest.cpp
-     # Classes/VRTest/VRTest.h
```

Comment:
On v3, i think you should modify project files too.

Reply:
ok